docs/deploying.md

```diff
@@ -130,13 +130,73 @@ Then you need to copy all your certificates and build the kustomization for
 ironic:
 
 ```sh
-   cp <path-to-ca-certificate> $BMOPATH/ironic-deployment/keepalived-tls/ca.crt
-   cp <path-to-tls-certificate> $BMOPATH/ironic-deployment/keepalived-tls/tls.crt
-   cp <path-to-tls-cert-key> $BMOPATH/ironic-deployment/keepalived-tls/tls.key
-   ./tools/bin/kustomize build $BMOPATH/ironic-deployment/keepalived-tls | \
+   cp <path-to-ca-certificate> $BMOPATH/ironic-deployment/tls/keepalived/ca.crt
+   cp <path-to-tls-certificate> $BMOPATH/ironic-deployment/tls/keepalived/tls.crt
+   cp <path-to-tls-cert-key> $BMOPATH/ironic-deployment/tls/keepalived/tls.key
+   ./tools/bin/kustomize build $BMOPATH/ironic-deployment/tls/keepalived | \
    kubectl apply -f -
 ```
 
+### Deploying with Basic Authentication
+
+You can deploy each of the alternatives above with basic authentication
+enabled. For this, you first need to follow the instructions of the above setup
+other than running kustomize.
```

Comment:
This looks pretty complicated. You could wrap it all up in a script (including generating a random password).